dotcom-rendering/src/components/NewsletterCard.tsx

```diff
@@ -71,13 +85,61 @@ export const buttonStyle = css`
 	}
 `;
 
-export const getButtonInitialAriaAttribute = (newsletterName: string) => ({
+const haveSeenMaker = css`
+	position: absolute;
+	bottom: 30;
+	right: 0;
+`;
+
+const getButtonInitialAriaAttribute = (newsletterName: string) => ({
 	'aria-label': `add ${newsletterName} to subscribe list`,
 	'data-aria-label-when-unchecked': `add ${newsletterName} to subscribe list`,
 	'data-aria-label-when-checked': `remove ${newsletterName} from subscribe list`,
 });
 
-export const NewsletterCard = ({ newsletter }: { newsletter: Newsletter }) => {
+export const NewsletterCard = ({
+	newsletter,
+	groupTitle,
+	cardPosition,
+	carouselPosition,
+}: Props) => {
+	const [hasBeenSeen, setRefForHaveSeenMarker] = useIsInView({});
```

Comment:
```suggestion
	const [hasBeenSeen, setRefForHaveSeenMarker] = useIsInView({
	    rootMargin: "0 -20px 0 20px"
	});
```

You can use Intersection observer properties here https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API

If you used a negative margin on the right side and a positive margin on the left side could you offset the intersection box so that the newsletter isn't marked as seen too early? And if you did that would you be able to avoid creating an extra span to apply the `hasBeenSeen` ref to?

Reply:
I've pushed a change to use the `threshold` property on the intersection observer and apply the ref to the container rather than using the marker element - it works well, thanks!